.travis.yml

```diff
@@ -1,110 +1,21 @@
 ---
 language: ruby
 
-rvm:
```

Comment:
Let's use the same Travis config as https://github.com/sensu/sensu-puppet/blob/master/.travis.yml

Only slightly different in testing for puppet-strings and some new minor releases of Puppet 5.

Reply:
Updated to use the linked .travis.yml.